server/mdm/android/service/service.go

```diff
@@ -13,11 +14,18 @@ import (
 	"github.com/fleetdm/fleet/v4/server/mdm/android"
 	"github.com/fleetdm/fleet/v4/server/mdm/android/service/proxy"
 	kitlog "github.com/go-kit/log"
+	"github.com/go-kit/log/level"
 	"google.golang.org/api/androidmanagement/v1"
 )
 
 // We use numbers for policy names for easier mapping/indexing with Fleet DB.
-const defaultAndroidPolicyID = 1
+const (
+	defaultAndroidPolicyID = 1
+	SignupSSESuccess       = "Android Enterprise successfully connected"
+)
+
+// SignupSSEInterval can be overwritten in tests.
+var SignupSSEInterval = 3 * time.Second
```

Comment:
Just a heads-up that package-level vars like this end up causing races in parallel tests, could it be e.g. a field on the `Service` type that is used for tests, and the default value is a const?